cmd/get/catalogs/printer.go

```diff
@@ -61,6 +61,14 @@ func (r *runner) printNoResourcesOutput() {
 	fmt.Fprintf(r.stdout, "  kubectl gs template catalog --help\n")
 }
 
+func getAppCatalogEntryDescription(description string) string {
+	if len(description) > 80 {
```

Comment:
should this be fixed or rather configurable via flag?

Reply:
Do you mean the character limit?
If you think there is a use case for it, feel free to add a flag and set the default to 80.

Reply:
I don't think its a typical use case but I could see a flag being useful for truncating or extending the output. 

@ljakimczuk I agree with @anvddriesch feel free to add if you think its needed.

Reply:
@rossf7 @anvddriesch what do you think about the current approach? Flag is persistent so it can be re-used with other get sub-commands. I don't like passing the argument down the stream to the function that need it though, it could be set as a global variable, but I am not sure what's the right style/approach 😅

Reply:
@ljakimczuk I would add it to flag.go like I described here.
https://github.com/giantswarm/kubectl-gs/pull/698/files#r821565602